org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -641,18 +641,6 @@ SsoLlmDeveloper:
     managedPolicies:
       - 'arn:aws:iam::aws:policy/AmazonBedrockFullAccess'
       - 'arn:aws:iam::aws:policy/AWSCloudFormationFullAccess'
-#   https://stackoverflow.com/questions/58125181/cloud-formation-cant-upload-template-file
-    inlinePolicy: >-
```

Comment:
Instead of removing this inline policy in this PR i would suggest you do this in a follow on PR that uses `SynapseLlmDeveloperPolicy`.  Make the follow on PR `depends on` this PR.  It will help provide reviewers with a complete picture of this change.

Reply:
OK